src/Style/General/ButtonStyle.ts

```diff
@@ -104,7 +73,7 @@ export const DefaultBtn = styled(Button)<DefaultBtnProps>`
       default:
         return `
           background-color: ${SecondaryColor.white};
-          color: ${SecondaryColor.black};
+          color: ${SecondaryColor.white};
```

Comment:
I think the color should be `black` here. If not in the `default` case, it will show an invisible button :sweat_smile:

Reply:
Changed it.